### ISSUE TITLE
Wrong pulseaudio path in init script ?

### ISSUE THREAD

**Issue by junekiwi** [BaReinhard/a2dp_bluetooth#3]
Hi,

In my installation, the pulseaudio binary was in `/usr/bin` and not in `/usr/local/bin`.
Maybe the init script should be updated ?

https://github.com/BaReinhard/a2dp_bluetooth/blob/0d2819d73fe9e88a26a12edf9b923e2a51d83845/a2dp_source/init.d/pulseaudio#L15

**Comment by BaReinhard** [comment]
Pulse5 is installed in /usr/bin, once pulse6 is compiled in the script there should also be pulseaudio in /usr/local/bin.

If that’s not the case can you show me the contents of 

`cat /etc/os-release`

**Comment by junekiwi** [comment]
I am on Stretch:
```bash
PRETTY_NAME="Raspbian GNU/Linux 9 (stretch)"
NAME="Raspbian GNU/Linux"
VERSION_ID="9"
VERSION="9 (stretch)"
ID=raspbian
ID_LIKE=debian
HOME_URL="http://www.raspbian.org/"
SUPPORT_URL="http://www.raspbian.org/RaspbianForums"
BUG_REPORT_URL="http://www.raspbian.org/RaspbianBugs"
```

But actually I realized the compilation of pulseaudio failed because of the absence of `intltool`, which is in turn missing because apt failed on `libjson0-dev`:
```bash
E: Unable to locate package libjson0-dev
```
https://github.com/BaReinhard/a2dp_bluetooth/blob/0d2819d73fe9e88a26a12edf9b923e2a51d83845/a2dp_source/configure.sh#L61


I relaunched the whole script with `libjson-c-dev` instead and it seems to work :-)

**Comment by BaReinhard** [comment]
Yeah this install is meant for Jessie and not stretch,  but if it is working that is great news.!

**Comment by Siewert308SW** [comment]
Then one question as I asked you earlier but somehow we never got beck on it.
If I'm right the BT device should auto reconnect, right. 
On Jessie is never reconnects, at least on my side.

**Comment by BaReinhard** [comment]
It should, it’s possible the device you are connecting doesn’t support that feature. What device are you trying to connect?

**Comment by Siewert308SW** [comment]
It connect and plays sound but when disconnected it doesn't auto reconnect and I have to connect it manual again.
Using a HEOS Denon BT Speaker

**Comment by BaReinhard** [comment]
Let me ask you this, when you connect your phone to the BT speakers and then turn off the speakers, then turn them back on, does it reconnect to your phone?

**Comment by BaReinhard** [comment]
Working on a stretch branch to merge with the master I believe all the code works but I haven’t tested yet. It should work, luckily with stretch it comes with pulse10 so no need to compile pulse, I believe some other compilations are also unnecessary so the install speed should increase for stretch.
